README.md

```diff
@@ -92,3 +92,27 @@ Best practice for managing a pull request
 
      ```make gh-publish```
 
+
+Use docker to build and preview the website
+--------------------------------------------
+To build CNERG website using docker you can use the cnerg container containing
+all the required dependencies to build the website cnerg/cenrg_XXXX.
+
+1. go into your folder hosting cnerg website sources
```

Comment:
for clarity it might be nice to add a couple of mock the git clone commands here. Some people using this may be new to the group and just getting familiar with the git commands too.

Reply:
I am not sure this should be part of the docker instruction, might need a get started section somewhere to setup and explain what you need to change to add yourself

Reply:
Then maybe this could read something like "from the directory containing the website repository" with a link to those instructions on cloning the site